src/api/app/models/project.rb

```diff
@@ -1689,7 +1689,8 @@ def status_reports(checkables)
   end
 
   def combined_status_reports
-    @combined_status_reports ||= status_reports(repositories) | status_reports(repository_architectures)
+    @combined_status_reports ||= status_reports(repositories.where.not(required_checks: nil)) |
```

Comment:
[nitpick] I would move `where.not(required_checks: nil)` to status_reports:

```
def status_reports(checkables)
    checkables = checkables.where.not(required_checks: nil)
```

so that it keep the same logic as before and don't duplicate code

Reply:
Or just drop it - if the caches work, it doesn't matter too much and avoids the problem with bootstraping stagings' reports.

Reply:
I already fixed the specs now :smile: